File: samples/client/3_1_0_unit_test/java/src/main/java/org/openapijsonschematools/client/components/schemas/ObjectTypeMatchesObjects.java
package org.openapijsonschematools.client.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.schemas.MapJsonSchema;
import org.openapijsonschematools.client.schemas.validation.FrozenMap;

public class ObjectTypeMatchesObjects extends MapJsonSchema {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class ObjectTypeMatchesObjects1 extends MapJsonSchema.MapJsonSchema1 {
        private static @Nullable ObjectTypeMatchesObjects1 instance = null;
        public static ObjectTypeMatchesObjects1 getInstance() {
            if (instance == null) {
                instance = new ObjectTypeMatchesObjects1();
            }
            return instance;
        }
    }

}

File: samples/client/3_1_0_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/ObjectTypeMatchesObjects.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.MapJsonSchema
import org.openapijsonschematools.client.schemas.validation.FrozenMap

class ObjectTypeMatchesObjects: MapJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class ObjectTypeMatchesObjects1 : MapJsonSchema.MapJsonSchema1() {
        companion object {
            @Volatile
            private var instance: ObjectTypeMatchesObjects1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: ObjectTypeMatchesObjects1().also { instance = it }
                }
        }
    }

}
